staff/acct/max-uid

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+"""Find the highest non-excluded OCF UID."""
+import sys
+
+from ocflib.account.creation import IGNORED_UID_RANGES
+from ocflib.account.creation import RESERVED_UID_RANGES
+from ocflib.infra.ldap import ldap_ocf
+from ocflib.infra.ldap import OCF_LDAP_PEOPLE
+
+
+INVALID_UID_RANGES = sorted(RESERVED_UID_RANGES + IGNORED_UID_RANGES)
+
+
+def is_valid_uid(uid):
+    for start, end in IGNORED_UID_RANGES:
+        if start <= uid <= end:
+            return False
+    return True
+
+
+def main():
+    print('Searching for maximum currently used OCF UID')
+    with ldap_ocf() as c:
+        c.search(
```

Comment:
Similar to what @cg505 said this is fine but indeed a little slow tbh; if we are not running this constantly I guess it's fine?